app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -106,7 +106,7 @@ def create_admin_set
       def setup_form
         add_breadcrumb t(:'hyrax.controls.home'), root_path
         add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-        add_breadcrumb t(:'hyrax.admin.sidebar.admin_sets'), hyrax.admin_admin_sets_path
+        add_breadcrumb t(:'hyrax.dashboard.my.action.edit_admin_set'), hyrax.admin_admin_sets_path
```

Comment:
You've changed the title here to say "Edit collection", but this goes to the index path.  Will that be confusing?

Reply:
Quite right - Expect an update to this PR.